frontend/src/scenes/trends/persons-modal/persons-modal-utils.tsx

```diff
@@ -17,14 +17,15 @@ import { InsightLabel } from 'lib/components/InsightLabel'
 import { getBarColorFromStatus, getSeriesColor } from 'lib/colors'
 
 export const funnelTitle = (props: {
-    step: number
+    converted: boolean
+    step?: number
```

Comment:
In what situation is this undefined?

Reply:
Hmmm I thought about this a little deeper. I was trying to keep the bad logic of before but now that this function is more "pure" the step can always just be given as it should be. Corrected 👍